helm/application-namespace-prep/values.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "properties": {
+        "authnK8s": {
+            "required": [
+                "goldenConfigMap",
+                "namespace"
+            ],
+            "properties": {
+                "goldenConfigMap": {
+                    "type": "string",
+                    "maxLength": 252,
+                    "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
```

Comment:
I left a comment in PR #257 re. the validation checks that we're adding for the Kubernetes names. I'm pretty sure these are not needed since there are built-in Kubernetes validation checks that apparently take precedence. You can confirm what I'm seeing by removing these checks and trying some bad values... the error messages that I see with this are the same whether our validation is in `values.schema.json` or not.